osu.Game/Screens/Play/HUD/HitErrorDisplay.cs

```diff
@@ -0,0 +1,198 @@
+                        {
+                            Anchor = Anchor.Centre,
+                            Origin = Anchor.Centre,
+                            RelativePositionAxes = Axes.Y,
+                            Icon = reversed ? FontAwesome.Solid.ChevronRight : FontAwesome.Solid.ChevronLeft,
+                            Size = new Vector2(8),
+                        }
+                    },
+                }
+            });
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)
+        {
+            config.BindWith(OsuSetting.ScoreMeter, type);
+        }
+
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+            HitWindows.SetDifficulty(beatmap.Value.BeatmapInfo.BaseDifficulty.OverallDifficulty);
+
+            bar.AddRange(new Drawable[]
+            {
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = ColourInfo.GradientVertical(colours.Yellow.Opacity(0), colours.Yellow),
+                    Height = (float)((HitWindows.Meh - HitWindows.Good) / (HitWindows.Meh * 2))
+                },
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colours.Green,
+                    Height = (float)((HitWindows.Good - HitWindows.Great) / (HitWindows.Meh * 2))
+                },
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colours.BlueLight,
+                    Height = (float)(HitWindows.Great / HitWindows.Meh)
+                },
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = colours.Green,
+                    Height = (float)((HitWindows.Good - HitWindows.Great) / (HitWindows.Meh * 2))
+                },
+                new Box
+                {
+                    RelativeSizeAxes = Axes.Both,
+                    Colour = ColourInfo.GradientVertical(colours.Yellow, colours.Yellow.Opacity(0)),
+                    Height = (float)((HitWindows.Meh - HitWindows.Good) / (HitWindows.Meh * 2))
+                }
+            });
+
+            type.BindValueChanged(onTypeChanged, true);
+        }
+
+        private void onTypeChanged(ValueChangedEvent<ScoreMeterType> type)
+        {
+            switch (type.NewValue)
+            {
+                case ScoreMeterType.None:
+                    this.FadeOut(200, Easing.OutQuint);
+                    break;
+
+                case ScoreMeterType.HitError:
+                    this.FadeIn(200, Easing.OutQuint);
+                    break;
+            }
+        }
+
+        public void OnNewJudgement(JudgementResult newJudgement)
+        {
+            if (!newJudgement.IsHit)
+                return;
+
+            Container judgementLine;
```

Comment:
looks beautiful for me
```suggestion
            var judgementLine = CreateJudgementLine(newJudgement);
```